src/utils.ts

```diff
@@ -27,19 +27,28 @@ interface AnyObject {
   [key: string]: any;
 }
 
+function transformArrayToObject(source: AnyObject) {
```

Comment:
Transform an array of objects into a single object. Risk here is duplicate keys in objects will collide in final object, where last one wins.